site/content/docs/browser-checks/timeouts.md

```diff
@@ -6,16 +6,29 @@ menu:
     parent: "Browser checks"
 ---
 
-All browser checks are capped at **120 seconds**. This means everything in your script needs to happen within those 120 seconds.  
+There are different kinds of timeouts you will encounter while working with Browser checks:
 
-## Dealing with timeouts
+| Timeout name          | Timeout origin | Default value | Can it be changed? |
+|-----------------------|----------------|---------------|--------------------|
+| Browser check timeout | Checkly        | 120 seconds   | No                 |
```

Comment:
Should we call these

- Browser Check execution timeout
- Playwright test timeout
- Playwright navigation timeout
- Playwright action timeout

🤔

Reply:
Done